coderedcms/blocks/html_blocks.py

```diff
@@ -81,6 +81,13 @@ class EmbedGoogleMapBlock(BaseBlock):
         label=_('Map zoom level'),
         help_text=_('Requires API key to use zoom. 1: World, 5: Landmass/continent, 10: City, 15: Streets, 20: Buildings')
     )
+    place_name = blocks.CharBlock(
+        required=False,
+        max_length=255,
+        label=_('Location name'),
+        help_text=_("Name of map location for html title attribute of map")
```

Comment:
I might avoid naming this `place_name` because Google uses "place" as special terminology in their API. It makes it feel like it's part of Google Maps. Maybe something like `map_title` (and corresponding label "Map title") would be more appropriate? I would also move this up underneath `search` so that they are grouped together, since the other `place_id` and `map_zoom_level` attributes are API-specific and rarely used.

For the help_text, maybe something a little less developer-y. "html title attribute" probably won't mean anything to content editors. Maybe something like: "Name of the map, for screen readers and accessibility purposes" <- not sure I actually like that phrasing, but something along those lines. Suggestions?

Reply:
changed to map_title for attribute name and label and adjusted help text, see edits